src/sentry/api/issue_search.py

```diff
@@ -240,6 +263,21 @@ def convert_search_filter(
                 operator = "IN" if search_filter.operator in EQUALITY_OPERATORS else "NOT IN"
             else:
                 operator = "=" if search_filter.operator in EQUALITY_OPERATORS else "!="
+
+            if search_filter.key.name == "substatus":
+                if not features.has("organizations:issue-states", organization):
+                    raise InvalidSearchQuery(
+                        "The substatus filter is not supported for this organization"
+                    )
+
+                status = GROUP_SUBSTATUS_TO_STATUS_MAP.get(
+                    new_value[0] if isinstance(new_value, list) else new_value
```

Comment:
Hmm, not sure if grabbing the first `substatus` value here would be good UX. Should we raise an error instead or let the query go through? 

Although, not sure if the `is:` query syntax even allows list values so we might be ok here.

Reply:
`new_value` is `[0]` when searching for `is:unresolved`. Not sure why.. 

Added error handling if the length is >1 though.